app/src/main/java/pl/example/android/garbageapp/ui/BaseActivitySector.java

```diff
@@ -25,4 +35,25 @@ protected void onCreate(Bundle savedInstanceState) {
         DetailViewModelFactory factory = InjectorUtils.provideDetailViewModelFactory(currentSector().getApplicationContext(), sectorType());
         mViewModel = ViewModelProviders.of(currentSector(), factory).get(DetailActivityViewModel.class);
     }
+
+    protected boolean checkIfSectorMarkedAsNotification(){
```

Comment:
I would change the name of the method to isMarkedForNotification().

Reply:
Good point. It was my bad day for finding names :)